src/test/integration/users/User.t.sol

```diff
@@ -268,6 +276,17 @@ contract User is Test {
         return _completeQueuedWithdrawal(withdrawal, false);
     }
 
+    /// @notice We set the proof generation start time to be after the timestamp that pod restaking is activated
+    /// We do this to prevent proofIsForValidTimestamp modifier from reverting
+    function activateRestaking() public {
+        pod.activateRestaking();
+        // To ensure oracle proof timestamp is greater than pod.mostRecentWithdrawalTimestamp
+        cheats.warp(block.timestamp + 10);
+        timeMachine.createSnapshot();
```

Comment:
Why do you snapshot here?

Reply:
TimeMachine will jump back before the timestamp that pod.activateRestaking() is called and then the `proofIsForValidTimestamp` modifier will revert. I added this here to work around that